packages/paste-website/src/pages/patterns/empty-states/index.mdx

```diff
@@ -0,0 +1,444 @@
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<content>
+
+<Breadcrumb>
+  <BreadcrumbItem to="/">Home</BreadcrumbItem>
+  <BreadcrumbItem to="/patterns">Patterns</BreadcrumbItem>
+</Breadcrumb>
+<Heading as="h1" variant="heading10">
+  {props.pageContext.frontmatter.title}
+</Heading>
+<Paragraph>{props.pageContext.frontmatter.description}</Paragraph>
+
+<Box as="dl" display="flex">
+  <Text as="dt" color="colorTextWeak" lineHeight="lineHeight30">
+    <Box width="size10">Status</Box>
+  </Text>
+  <Text as="dd">beta</Text>
+</Box>
+
+</content>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Ingredients
+
+<Grid gutter="space30" marginBottom="space80">
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="../components/card">Card</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="../components/heading">Heading</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="../components/paragraph">Paragraph</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+</Grid>
+<Grid gutter="space30" marginBottom="space80">
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="../components/button">Button (optional)</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="../components/anchor">Anchor (optional)</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="../illustrations">Illustrations (optional)</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+</Grid>
+
+```jsx
+// import all ingredients for the empty state patterns
+import {​Card} from '@twilio-paste/core/card';
+import {Heading} from '@twilio-paste/core/heading';
+import {Paragraph} from '@twilio-paste/core/paragraph';
+import {Button} from '@twilio-paste/core/button';
+import {​Anchor} from '@twilio-paste/core/anchor';
+```
+
+## Usage
+
+### General
+
+This pattern is used to explain to a customer what they should do next if content is either unavailable or can't be displayed.
+
+Empty states should:
+
+- **Be clear:** If there's a path forward, explain it as concisely as possible. Include an explanation for why a customer is seeing the empty state if it helps them better understand what to do next.
+- **Give direction and educate the customer:** Let customers know what steps they can take to move forward in a flow. Guiding them to "Create an alpha sender" is more helpful than saying "No alpha senders".
```

Comment:
What does "in a flow" mean when we say "move forward in a flow"? I was confused by it and would suggest removing it and having the sentence read "Let customers know what steps they can take to move forward."

Reply:
I think it's referring to a user flow. I like your suggestion. Thanks!